src/components/form/form.sandbox.ts

```diff
@@ -23,7 +25,7 @@ export class MFormSandbox extends ModulVue {
     ];
     serverResponse: any = this.serverResponses[0];
     hasServerResponse: boolean = false;
-    changeWatcher: any = {};
+    observedChange: any = '';
```

Comment:
ça serait possible de laisser les types?

Reply:
Je connais pas bien rxjs mais voici mon essai pour vous aider ce qui semble fonctionner au niveau du compilateur

`
this.forms[17].Changes
            .pipe(
                debounce(() => timer(500)),
                skipWhile((v: FormChange) => v.value.length < 3),
                map((v: FormChange) => `A mapped value ${v.value} of the change for field ${v.field}`),
                tap(() => this.$log.info('streamed data is mapped'))
            )
            .subscribe((v: string) => this.observedChange = v);
`